editor/src/components/inspector/sections/layout-section/flex-container-subsection/flex-container-controls.tsx

```diff
@@ -264,9 +271,61 @@ export const FlexGapControl = React.memo(() => {
     'FlexGapControl flexDirection',
   )
 
+  const flexGapControlsForHoverAndFocused: {
+    hovered: Array<CanvasControlWithProps<SubduedFlexGapControlProps>>
+    focused: Array<CanvasControlWithProps<SubduedFlexGapControlProps>>
+  } = React.useMemo(
+    () => ({
```

Comment:
This can be moved to the module scope, and then the `useMemo` won't be necessary